Float.FileDownloader.Tests/Float.FileDownloader.Tests.csproj

```diff
@@ -1,24 +1,41 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
```

Comment:
@bbeversdorf Does the tests project need to target multiple frameworks?

Reply:
If we want to ensure it runs against the proper target then we should be targeting multiple frameworks. You could make an argument to drop `net6`

Reply:
@bbeversdorf Should we specify `netstandard2.1` here instead? (Which covers .NET core 3.0 and higher?)

Reply:
Or...since .NET Standard is technically deprecated, maybe it is better to explicitly list out the .NET/.NET Core versions?
(Open either way.)

Reply:
@mediabounds .NET Standard is the API. It is not the actual framework.
https://xunit.net/docs/why-no-netstandard

Reply:
@bbeversdorf I know--but `netstandard2` is the target framework [of the actual project itself](https://github.com/gowithfloat/Float.FileDownloader/blob/5433fdae65852018301891365853b7255ffd1845/Float.FileDownloader/Float.FileDownloader.csproj#L3) -- so it can be the target framework. But! That article still answers the question which is that xUnit chooses to be more specific in the frameworks it targets. So my question is answered!